packages/votingV2/src/mappings/votingV2.ts

```diff
@@ -316,9 +317,9 @@ function processSlashesInSameTransaction(
   voteSlashed: SlashedVote,
   oldUserCalculatedStake: BigDecimal,
   votingContract: VotingV2,
-  requestIndex: BigInt
+  resolvedPriceRequestIndex: BigInt
```

Comment:
i feel like the variable naming here has become a bit inconsistent. there was an effort to remove "price" from events, but yet it was explicitly added here. are we dealing with prices or not

Reply:
I see your point. This uses the same terminology that we use in [VotingV2](https://github.com/UMAprotocol/protocol/blob/master/packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol#L90)